src/solver.jl

```diff
@@ -41,6 +41,9 @@ Create a solver, where the summation-by-parts (SBP) operators are of order `accu
 associated to the `mesh`.
 """
 function Solver(mesh, accuracy_order)
+    if isodd(accuracy_order)
+        @warn "DispersiveShallowWater is suspecting a central difference operator. This is not given for an odd accuracy order.\n This can lead to significant reduction in the order of convergence of the solution."
```

Comment:
```suggestion
        @warn "DispersiveShallowWater is expecting a central difference operator. This is not given for an odd accuracy order.\n This can lead to significant reduction in the order of convergence of the solution."
```
"suspect" sounds so negative :sweat_smile: